### PR TITLE
fix(curriculum): Remove unnecessary assert message argument from English challenges JavaScript Algorithms and Data Structures - 01

### PR DESCRIPTION
**Pull request by RandellDawson**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x]  My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Related to issue #36400

This PR removes assert message arguments from `testString` for the following sub section of the `JavaScript Algorithms and Data Structures` curriculum section:
* Basic JavaScript

### CODE REVIEW

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1:

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/introducing-else-if-statements.english.md [hunk 1]
```diff
@@ -32,21 +32,21 @@ Convert the logic to use <code>else if</code> statements.
...
 tests:
   - text: You should have at least two <code>else</code> statements
-    testString: assert(code.match(/else/g).length > 1, 'You should have at least two <code>else</code> statements');
+    testString: assert(code.match(/else/g).length > 1);
   - text: You should have at least two <code>if</code> statements
-    testString: assert(code.match(/if/g).length > 1, 'You should have at least two <code>if</code> statements');
-  - text: You should have closing and opening curly braces for each condition
-    testString: assert(code.match(/if\s*\((.+)\)\s*\{[\s\S]+\}\s*else if\s*\((.+)\)\s*\{[\s\S]+\}\s*else\s*\{[\s\S]+\s*\}/), 'You should have closing and opening curly braces for each condition in your if else statement');
+    testString: assert(code.match(/if/g).length > 1);
+  - text: You should have closing and opening curly braces for each condition <code>else if</code> code block.
```
Comment (thecodingaviator) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/introducing-else-if-statements.english.md]: Has something been changed here? I think this is better:
```suggestion
  - text: You should have closing and opening curly braces for each <code>if else</code> code block.
```

**Review by thecodingaviator — COMMENTED** [review]
Just worried about the one above, rest LGTM

**Comment by RandellDawson** [comment]
@thecodingaviator I made the word change in the text you suggested.

**Review by thecodingaviator — APPROVED** [review]
LGTM

**Comment by thecodingaviator** [comment]
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 📝

**Pull request merged**